fastlane_core/spec/core_ext/shellwords_ext_spec.rb

```diff
@@ -0,0 +1,372 @@
+# references
```

Comment:
Should this file actually be called `shellwords_spec.rb` to match with the new `shellwords.rb`?

Reply:
I chose the name after the other file `\fastlane_core\spec\core_ext\cfpropertylist_ext_spec.rb` which contains the tests for an implementation in `\fastlane_core\lib\fastlane_core\core_ext\cfpropertylist.rb`. But the other one would work as well, of course.